app/src/main/java/com/coveros/coverosmobileapp/website/MainActivity.java

```diff
@@ -3,22 +3,34 @@
 import android.app.AlertDialog;
 import android.content.Context;
 import android.content.DialogInterface;
+import android.content.Intent;
 import android.net.ConnectivityManager;
 import android.net.NetworkInfo;
-import android.os.Bundle;
+import android.support.v4.widget.DrawerLayout;
 import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
 import android.webkit.WebSettings;
 import android.webkit.WebView;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.ListView;
 import android.widget.Toast;
 
 import com.coveros.coverosmobileapp.R;
+import com.coveros.coverosmobileapp.blogpost.BlogPostsListActivity;
+
 
 @SuppressWarnings("squid:MaximumInheritanceDepth")
 public class MainActivity extends AppCompatActivity {
     //MainActivity
     private String webName;
     private WebView browser;
 
+    private static final String[] menuTitles = new String[]{"Website","Blog","Bookmarks"};
```

Comment:
This is exactly the same list of menu titles is defined here and in BlogPostListActivities.java. If that is not a coincidence (and I'm guessing it isn't) then they should be defined in one place. Also, the previous suggestion regarding capitalization of constant names applies here as well.